Source/player.h

```diff
@@ -305,18 +305,25 @@ struct PlayerStruct {
 	Uint8 *_pDData;
 	Uint8 *_pBData;
 
+	/**
+	 * @brief Search the players inventory for a given item.
+	 * @param The item idx identifyer
+	 * @return The inventory index of the item, or -1 if not found.
+	 */
+	int GetItemIndex(int idx) const;
+
 	/**
 	 * @brief Gets the base value of the player's specified attribute.
 	 * @param attribute The attribute to retrieve the base value for
 	 * @return The base value for the requested attribute.
-	*/
+	 */

```

Comment:
I believe this reformatting is incorrect. Standard doxygen has it in the beggining, like this:

![image](https://user-images.githubusercontent.com/461579/114289905-da464980-9a51-11eb-8235-7a762d9059b9.png)

Reply:
No
![image](https://user-images.githubusercontent.com/204594/114290395-b7328e80-9a7f-11eb-8ee2-e883c89b6aaa.png)

Reply:
This will be a problem... Visual Studio Doxygen format generates it the way I shared :S